spec/patches.md

```diff
@@ -55,6 +55,26 @@ Modify step 2.1 to read:
       [=implementation-defined=] manner.
     * |schedulerQueue|'s [=scheduler task queue/tasks=], if |schedulerQueue| is not null.
 
+Note: the HTML specification enables per-[=task source=] prioritization by making the selection of
+the next [=task=]'s [=task queue=] in the event loop processing steps [=implementation-defined=].
+Similarly, this specification makes selecting between the next {{Scheduler}} task and the next task
+from an [=event loop=]'s [=task queues=] [=implementation-defined=], which provides UAs with the
+most scheduling flexibility.
+<br/><br/>
+But the intent of this specification is that the {{TaskPriority}} of {{Scheduler}} tasks would
+influence the event loop priority. Specifically, "{{TaskPriority/background}}" tasks and
```

Comment:
This sentence is still a bit hard to parse, with a number of "and"s and "as well as". Maybe a small change could help:

> Specifically, "background" tasks and continuations are typically considered less important than most other event loop tasks, _**while**_ "user-blocking" tasks and continuations, as well as "user-visible" continuations (but not tasks), are typically considered to be more important.